Mention core-designsystem module in the modularization journey docs (#755)
android/nowinandroid: docs/ModularizationLearningJourney.md

   <td><code>TopicsRepository</code><br>
   </td>
  </tr>
  <tr>
   <td><code>core:designsystem</code>
   </td>

>    <td>Design system, tokens and icons.

Reviewer: ```suggestion
   <td>Design system which includes Core UI components (many of which are customized Material 3 components), app theme and icons. 
```

   </td>
   <td><code>NiaTheme</code><br>

>    <code>NiaIcons</code>

Reviewer: ```suggestion
   <code>NiaIcons</code>    <code>NiaButton</code>    <code>NiaTheme</code> 
```

>    <td>Design system which includes Core UI components (many of which are customized Material 3 components), app theme and icons. The design system can be viewed by running the `app-nia-catalog` run configuration`. 

Reviewer: ```suggestion
   <td>Design system which includes Core UI components (many of which are customized Material 3 components), app theme and icons. The design system can be viewed by running the `app-nia-catalog` run configuration. 
```

>    <td><code>NiaTheme</code><br>

Reviewer: ```suggestion
   <td>
```

   </td>
  </tr>
  <tr>
   <td><code>core:ui</code>
   </td>

>    <td>UI components, composables and resources, used by different features.

Reviewer: ```suggestion
   <td>Composite UI components and resources used by feature modules, such as the news feed. Unlike the `designsystem` module, it is dependent on the data layer since it renders models, like news resources. 
```

   </td>

>    <td><code>NewsResourceCardExpanded</code>

Reviewer: ```suggestion
   <td> <code>NewsFeed</code> <code>NewsResourceCardExpanded</code>
```
